src/main/java/org/candlepin/insights/jmx/RhsmConduitJmxBean.java

```diff
@@ -53,4 +65,43 @@ public void syncOrg(String orgId) {
             log.error("Error during JMX-initiated sync for org ID {}", orgId, e);
         }
     }
+
+    @ManagedOperation(description = "Sync all orgs from the configured org list")
+    public void syncFullOrgList() {
+        log.info("Starting JMX-initiated sync for all configured orgs");
+        try {
+            tasks.syncFullOrgList();
+        }
+        catch (Exception e) {
+            log.error("Error during JMX-initiated sync for full org list", e);
+        }
+    }
+
+    @ManagedOperation(description = "Add some orgs to the database sync list")
+    @ManagedOperationParameter(name = "orgs", description = "comma-separated org list (whitespace ignored)")
+    public void addOrgsToSyncList(String orgs) {
+        List<Organization> orgList = extractOrgList(orgs);
+
+        log.info("Adding {} orgs to DB sync list", orgList.size());
+
+        repo.saveAll(orgList);
+    }
+
+    @ManagedOperation(description = "Remove some orgs from the database sync list")
+    @ManagedOperationParameter(name = "orgs", description = "comma-separated org list (whitespace ignored)")
+    public void removeOrgsFromSyncList(String orgs) {
+        List<Organization> orgList = extractOrgList(orgs);
+
+        log.info("Removing {} orgs from DB sync list", orgList.size());
+
+        repo.deleteAll(orgList);
+    }
+
```

Comment:
Would it make sense to add something that allows us to check if an org is in the list?

Reply:
Done

Reply:
Done